hr_family/models/hr_employee.py

```diff
@@ -36,3 +15,62 @@ class HrEmployee(models.Model):
     fam_mother = fields.Char("Mother's Name")
     fam_mother_date_of_birth = fields.Date(
         "Date of Birth", oldname='fam_mother_dob')
+    # you can't marry a company, corporations aren't people
+    partner_spouse_id = fields.Many2one(
+        'res.partner', string='Spouse', domain=[('is_company', '=', False)]
+    )
+    marital_status_id = fields.Many2one(
+        'hr.employee.marital.status', string='Marital status',
+    )
+
+    @api.constrains('partner_spouse_id')
+    def _check_spouse_uniqueness(self):
+        for this in self:
+            spouse = self.search(
+                [('partner_spouse_id', '=', this.partner_spouse_id.id),
```

Comment:
what happens if there is no spouse, but another employee without a spouse?

Reply:
https://github.com/OCA/hr/commit/3722b8005ad3a28827b89dbe0563bdeb77be68fa#diff-9815fc7e7d4f54c5c9e87c6bd11aa486R29